### ISSUE TITLE
Добавить правило запрещающее создание юнитов через методы домена

### ISSUE THREAD

**Issue by SQReder** [effector/eslint-plugin#135]
В связи с [переходом на новый](https://t.me/sergeysova/842) основной способ указания домена для создаваемых юнитов предлагаю завести соответствующее правило, чтобы поддержать консистентность

```ts
const domain = createDomain()

// Bad
const event = domain.createEvent()

// Good
const event = createEvent({ domain })
```

**Comment by igorkamyshev** [comment]
Отличная идея 💡
